playbooks/common/openshift-cluster/openshift_prometheus.yml

```diff
@@ -0,0 +1,12 @@
+---
+- include: evaluate_groups.yml
```

Comment:
This needs to be `std_include.yml` which also does `evaluate_groups.yml`.   We are moving to requiring all entry point playbooks to flow through the `std_include.yml` to ensure proper initialization.